src/omv/common/sensor.h

```diff
@@ -175,6 +175,8 @@ typedef struct _sensor {
     pixformat_t pixformat;      // Pixel format
     framesize_t framesize;      // Frame size
     int framerate;              // Frame rate
+    uint32_t last_frame_ms;     // Last sampled frame timestamp in milliseconds.
+    bool last_frame_ms_valid;   // Last sampled frame timestamp in milliseconds valid.
```

Comment:
```suggestion
```
Why not check if last_frame_ms is 0 ?

Reply:
Wanted to make it as robust as possible. 0 is a valid time.